content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -45,6 +45,68 @@ For a full list of supported libraries and processor architectures, see [Compati
 
 ## Installation and getting started
 
+### Configure the Datadog Agent for APM
+
+Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links below to enable trace collection within the Datadog Agent.
```

Comment:
"to receive traces from your now instrumented application" sounds very forced grammatically, and doesn't make sense anymore if we configure the Agent before the Tracer.

Reply:
"in" vs "within"

```suggestion
Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links below to enable trace collection in the Datadog Agent.
```